www/templates/blog.tx

```diff
@@ -1,12 +1,12 @@
 <form class="Submissions" action="/post/save" method="POST">
-    Title *<br /><input class="cooltext" type="text" name="title" placeholder="Iowa Man Destroys Moon" />
+    Title *<br /><input required class="cooltext" type="text" name="title" placeholder="Iowa Man Destroys Moon" />
```

Comment:
Let's remove the * here and on the other fields since the required thing should do the trick by itself